### ISSUE TITLE
Customisable highlighting — request

### ISSUE THREAD

**Issue by alexeiva** [googlei18n/fontdiff#11]
Is there a way to customize highlighting?

I would like to see the old font in gray, and the new font in black

![screen shot 2016-10-19 at 20 05 46](https://cloud.githubusercontent.com/assets/16634089/19544960/740bd3ce-9637-11e6-9dbf-e0601eb0288e.png)

**Comment by brawer** [comment]
Cool idea. Send a pull request?

**Comment by alexeiva** [comment]
Yes. Could you point me to the the piece of code responsible for that?

This would be very helpful in our GF Excellence workflow. 
Here are sample images describing this use case by @ThomasJockin

![mavenimprovement_08_10_2016](https://cloud.githubusercontent.com/assets/16634089/19545169/1a71baac-9639-11e6-9748-d66f76d9f3c5.png)
![mavenimprovement_08_15_2016](https://cloud.githubusercontent.com/assets/16634089/19545172/1b468c8c-9639-11e6-9915-b4c6b26daae8.png)
![screen shot 2016-10-19 at 20 17 38](https://cloud.githubusercontent.com/assets/16634089/19545174/1cdd25ce-9639-11e6-961e-92192fc90e61.png)

**Comment by brawer** [comment]
Here’s the current code for rendering a line if it has any deltas:
https://github.com/googlei18n/fontdiff/blob/master/src/fontdiff/paragraph.cc#L160

Feel free to add a mode where it emits output like you described. To get the desired visual effect, you can probably use translucent colors by calling [cairo-set-source-rgba](https://www.cairographics.org/manual/cairo-cairo-t.html#cairo-set-source-rgba). Good luck, and thanks in advance for doing it — it sounds quite useful indeed!
